blockchain/pool.go

```diff
@@ -28,7 +28,7 @@ var peerTimeoutSeconds = time.Duration(15) // not const so we can override with
 	Every so often we ask peers what height they're on so we can keep going.
 
 	Requests are continuously made for blocks of higher heights until
-	the limits. If most of the requests have no available peers, and we
+	we reach the limits. If most of the requests have no available peers, and we
```

Comment:
I thought you preferred the passive voice to the first person (I or we).

"until the limit is reached"?